osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/CorrelatedObject.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands;
```

Comment:
This class has been added to the code of the protocol adapter. It does not have anything specific to the protocol adapter though. It is useful for code that needs to do something with the correlation UID, like passing it on in calls to other services like the secret management service.
Since the secret management service is intended for use in other situations than the DLMS protocol adapter, and passing on the correlationUid could become desirable for other services as well, it looks like it would be a good idea to put this code in one of the shared projects, so it is available outside the protocol adapter.

Reply:
Not using CorrelatedObject anymore